app/assets/javascripts/webpush.js.erb

```diff
@@ -0,0 +1,47 @@
+<% ENV["VAPID_PUBLIC_KEY"] = ENV["VAPID_PUBLIC_KEY"] ||
+  "BEdmrrlB8FEHyvVKkzLaUPTQpIIPjqYB9A8fitbhDz-ZBgpbY3wYL1VY9M0Kz3vPVbRRQONpTXGYIioKullYZf4="
```

Comment:
Can we move this to config/environments/development.rb ?

Reply:
It is required for the tests I think

Reply:
Then you can add it in config/environments/test.rb as well